recipes/cppitertools/all/conanfile.py

```diff
@@ -64,7 +64,10 @@ def source(self):
 
     def package(self):
         copy(self, "LICENSE.md", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
-        copy(self, "*.hpp", src=self.source_folder, dst=os.path.join(self.package_folder, "include", "cppitertools"), excludes=('examples/**', 'test/**'))
+        if Version(self.version) < "2.2":
```

Comment:
I wonder if in order to avoid this, it might be a good opportunity to use the `CMakeLists.txt` from upstream instead? (not required for this PR if the right files are in the right place, but just as a comment)